apps/frontend/src/components/Composite/booking/sessions/ExpandedSessionCard/ExpandedSessionCard.tsx

```diff
@@ -1,25 +1,35 @@
 import type { CartGameSession } from "@/types/game-session"
+import { Box, Card, CardHeader, Text, VStack } from "@yamada-ui/react"
 import { LevelSelector } from "./LevelSelector"
-
 interface ExpandedSessionCardProps {
   gameSession: CartGameSession
 }
 
 export const ExpandedSessionCard = ({ gameSession }: ExpandedSessionCardProps) => (
-  <div className="rounded text-sm">
-    <div className="rounded-t-md bg-primary px-6 py-4 drop-shadow-lg">
-      <p className="font-medium text-lg text-primary-foreground">{gameSession.weekday}</p>
-      <p className="font-medium text-primary-foreground/70">{gameSession.locationName}</p>
-    </div>
-    <div className="bg-secondary p-6">
-      <p className="font-semibold text-tertiary">Address</p>
-      <p className="text-tertiary/70">{gameSession.locationAddress}</p>
+  <Card fontSize="sm" rounded="md" shadow="none">
+    <CardHeader backgroundColor="primary" paddingX={6} paddingY="md" roundedTop="md" shadow="lg">
+      <VStack gap="none">
+        <Text color="white" fontSize="lg" fontWeight="medium">
+          {gameSession.weekday}
+        </Text>
+        <Text color="whiteAlpha.700" fontWeight="medium">
+          {gameSession.locationName}
+        </Text>
+      </VStack>
+    </CardHeader>
+    <Box backgroundColor="gray.50" padding={6}>
+      <Text color="tertiary" fontWeight="semibold">
+        Address
+      </Text>
+      <Text color="tertiary.300">{gameSession.locationAddress}</Text>
       <br />
-      <p className="font-semibold text-tertiary">Time</p>
-      <p className="text-tertiary/70 uppercase">
+      <Text color="tertiary" fontWeight="semibold">
+        Time
+      </Text>
+      <Text casing="uppercase" color="tertiary.300">
         {gameSession.startTime} - {gameSession.endTime}
-      </p>
-    </div>
+      </Text>
+    </Box>
     <LevelSelector id={gameSession.id} selectedLevel={gameSession.playLevel} />
-  </div>
+  </Card>
```

Comment:
I think it's cleaner to implement it this way.
`gap` should be `0` instead of `none`.
making text color `whiteAlpha.800` for location name improves contrast.
You should use `<CardBody>` instead of `Box`.
```suggestion
  <Card bg="gray.50" fontSize="sm" variant="solid">
    <CardHeader backgroundColor="primary" paddingY="md" roundedTop="md">
      <VStack gap="0">
        <Text color="white" fontSize="lg">
          {gameSession.weekday}
        </Text>
        <Text color="whiteAlpha.800">{gameSession.locationName}</Text>
      </VStack>
    </CardHeader>
    <CardBody gap="sm">
      <VStack gap="xs">
        <Text color="tertiary" fontWeight="semibold">
          Address
        </Text>
        <Text color="tertiary.400">{gameSession.locationAddress}</Text>
      </VStack>
      <VStack gap="xs">
        <Text color="tertiary" fontWeight="semibold">
          Time
        </Text>
        <Text casing="uppercase" color="tertiary.400">
          {gameSession.startTime} - {gameSession.endTime}
        </Text>
      </VStack>
    </CardBody>
    <LevelSelector id={gameSession.id} selectedLevel={gameSession.playLevel} />
  </Card>
```